wherehows-web/app/components/dataset-compliance.ts

```diff
@@ -820,11 +824,12 @@ export default class DatasetCompliance extends Component {
         [TagFilter.showAll]: identity,
         [TagFilter.showReview]: tagsRequiringReview(complianceDataTypes, {
           checkSuggestions: false,
-          suggestionConfidenceThreshold
+          suggestionConfidenceThreshold: 0
         }),
         [TagFilter.showSuggested]: arrayFilter((tag: IComplianceChangeSet) =>
           tagSuggestionNeedsReview({ ...tag, suggestionConfidenceThreshold })
```

Comment:
Just wondering, shouldn't hey have the same value?

Reply:
which values should be the same?

Reply:
Ah, yea, it's an parameter to filter out suggestions not meeting a threshold and has no import when the `checkSuggestions` flag is false,  but the interface does not make it optional, maybe I should